torch/_subclasses/functional_tensor.py

```diff
@@ -56,6 +56,7 @@ class FunctionalTensor(torch.Tensor):
         torch.ops.aten.numel.default,  # type: ignore[has-type]
         torch.ops.aten.sym_numel.default,  # type: ignore[has-type]
         torch.ops.aten.dim.default,  # type: ignore[has-type]
+        torch.ops.prim.device.default,  # type: ignore[has-type]
```

Comment:
tbh I'm still curious how we ended up with a call to `prim.device` inside of `FunctionalTensorMode` (in my head, calls to `x.device` should always dispatch to `aten.device`, not `prim.device`)

Reply:
https://github.com/pytorch/pytorch/pull/113728#discussion_r1427432430